reactor-netty-http/src/main/java/reactor/netty/http/client/HttpClient.java

```diff
@@ -1210,6 +1210,10 @@ else if (configuration().metricsRecorder() != null) {
 		}
 	}
 
+	public HttpClient metrics(boolean enable, String prefix) {
+		return metrics(enable, () -> new MicrometerHttpClientMetricsRecorder(prefix));
+	}
+
```

Comment:
Is this needed? One can use 
- `reactor.netty.http.client.HttpClient#metrics(boolean, java.util.function.Supplier<? extends reactor.netty.channel.ChannelMetricsRecorder>)`
- `reactor.netty.http.client.HttpClient#metrics(boolean, java.util.function.Supplier<? extends reactor.netty.channel.ChannelMetricsRecorder>, java.util.function.Function<java.lang.String,java.lang.String>)` with `MicrometerHttpClientMetricsRecorder`